lib/app/routes/exercises.rb

```diff
@@ -10,7 +10,9 @@ class Exercises < Core
           redirect '/inbox'
         end
 
-        inbox = ::Inbox.new(current_user, session[:inbox], session[:inbox_slug])
+        inbox = ::Inbox.new(current_user,
+                            session[:inbox] || current_user.default_language,
+                            session[:inbox_slug] || current_user.default_slug)
```

Comment:
We don't need a default for the slug. If there's no slug then the default list will be everything they have access to within the default language.